src/node.ts

```diff
@@ -85,7 +85,12 @@ export class EthereumNode extends constructs.Construct {
 
     super(scope, id);
 
+
     const region = cdk.Stack.of(this).region;
+    const availabilityZone = cdk.Stack.of(this).availabilityZones;
+    // In order to support Environment Agnostic Stacks (https://docs.aws.amazon.com/cdk/v2/guide/environments.html),
+    // we need to check whether or not the region is set as an environment variable in the CDK app.
+    var tokenRegion = cdk.Token.isUnresolved(region);
```

Comment:
Let's be a bit more descriptive here with something like:

```
const regionInEnvironment = !cdk.Token.isUnresolved(region);
```

Reply:
Changed!